annoying/tests/fields.py

```diff
@@ -1,8 +1,42 @@
+import json
+
 from django.test import TestCase
 from . import models
 
 
+def dump_dict(d):
+    """
+    Returns a JSON string sorted by keys
+    """
+    return json.dumps(d, sort_keys=True)
+
+
 class FieldsTestCase(TestCase):
     def test_auto_one_to_one(self):
         super_villain = models.SuperVillain.objects.create()
         self.assertEqual(super_villain.mortal_enemy.name, "Captain Hammer")
+
+    def test_json_field_create(self):
+        stats = {
+            'strength': 100,
+            'defence': 50,
+        }
+        super_villain = models.SuperVillain.objects.create(stats=stats)
+        self.assertEqual(super_villain.stats['strength'], 100)
```

Comment:
These tests don't do much, they just read the values in `stats` without loading them from the DB, but they don't hurt either. You have the proper tests below, so that's good.